2-factor-authentication/test/helper/jest.setup.ts

```diff
@@ -16,34 +16,28 @@ beforeAll(async () => {
      * Select the display you want to run your tests on, display 0 is your main display;
      * ignore if you have only one display
      */
-    display: 2,
-    port: 7007
+    //display: 1, enable it only when you have more then one display
+    port: 7007,
+
   });
 
   await uiController.start();
 
   aui = await UiControlClient.build({
-    credentials: {
-      workspaceId: '<your workspace id>',
-      token: '<your access token>',
-    },
-    uiControllerUrl:'http://127.0.0.1:7007'
+    uiControllerUrl: 'http://127.0.0.1:7007'
   });
 
   auiAndroid = await UiControlClient.build({
-    credentials: {
-      workspaceId: '<your workspace id>',
-      token: '<your access token>',
-    },
-    uiControllerUrl:'http://127.0.0.1:6769'
+    uiControllerUrl: 'http://127.0.0.1:6769',
+    'annotationLevel': AnnotationLevel.ON_FAILURE
```

Comment:
let's rm quotation marks around keys if not necessary

Is this annotation level at all necessary or wasn't it simply meant for debugging?

Reply:
I will remove it, it might be useful but only if you knew about it